src/api/app/views/webui2/webui/user/register_user.html.haml

```diff
@@ -0,0 +1,12 @@
+:ruby
+  @pagetitle = 'Create User'
+
+.card
+  .card-body
+    .row
+      .col-lg-6
+        - if can_register
+          %h5.card-title  Create User
+          = render partial: 'webui2/shared/sign_up'
```

Comment:
The submit button on this view is still `Sign Up`. It should be `Create`. Passing a variable to the partial for this would be a way to do it.